index.html

```diff
@@ -9,11 +9,11 @@
             <img src="/programming/web/projects/Axitproject/images/logo.png" alt="Logo">
```

Comment:
The image is not loaded on my computer. I believe the image src is wrong.
You are using a path relative to the file system root (see => `/programming/web/...`).
You need to use the path relative to the current folder.

![image](https://user-images.githubusercontent.com/13898415/144059079-4347ac56-aee3-4dea-8e85-4c4947a01d14.png)


Read this https://www.w3schools.com/html/html_filepaths.asp